ui/components/MesheryPerformanceComponent.js

```diff
@@ -111,17 +111,43 @@ class MesheryPerformanceComponent extends React.Component {
     };
   }
 
-  handleChange = (name) => (event) => {
-    if (name === 'url' && event.target.value !== '') {
-      this.setState({ urlError: false });
-    }
-    if (name === 't' && (event.target.value.toLowerCase().endsWith('h')
-      || event.target.value.toLowerCase().endsWith('m') || event.target.value.toLowerCase().endsWith('s'))) {
-      this.setState({ tError: false });
-    }
+  // handleChange = (name) => (event) => {
```

Comment:
Why did you comment out this code?

Reply:
Actually I forgot to clear out that code. I will remove that. The edits which I have done are the replacement of the commented code.